### ISSUE TITLE
overlay text and call to action buttons on banners

### ISSUE THREAD

**Issue by mbyczkowski** [mbyczkowski/lacocina#8]
This one is a bit tricky. Basically, the 'text' and 'button' where initially just part of the image (I was lazy and took a screenshot of the design), but then I started building something like an 'overlay'. The tricky thing is as the screen gets smaller (say on mobile) this text has to make sense (and the image will probably have to change, too). Maybe we could even build some helper methods to make creation of those banners easier? (again, I don't know how many of those we'll have across the website)

**Comment by katiemthom** [comment]
<img width="1130" alt="screen shot 2016-09-04 at 10 01 01 am" src="https://cloud.githubusercontent.com/assets/4976125/18232560/8f9e9592-7286-11e6-9d6e-ab428af9812f.png">

Is this what you're referring to? I'm not sure exactly what you have in mind with this issue. The banner/text/CTA seems to be responsive.

**Comment by katiemthom** [comment]
I decided to put in a bit more effort here and I think I found what you're referencing:

Home page:
<img width="1101" alt="screen shot 2016-09-05 at 7 34 36 am" src="https://cloud.githubusercontent.com/assets/4976125/18250998/6f352ce8-733b-11e6-863c-bd7b10ee1972.png">

Volunteer and catering pages:
<img width="1087" alt="screen shot 2016-09-05 at 7 36 25 am" src="https://cloud.githubusercontent.com/assets/4976125/18251013/7ece23e4-733b-11e6-97a3-c59f4f3393fa.png">

So we want an easy way to overlay text + CTA on a large image and have that be automatically responsive. I can get to work on something basic. @abyczkowska let me know if you have anything specific in mind for the design and @mbyczkowski let me know if this isn't what you had in mind.

**Comment by abyczkowska** [comment]
@katiemthom those images are incorrect. Zeplin has the most updated mockups.

**Comment by mbyczkowski** [comment]
@katiemthom Yes! This is exactly what I had in mind. It's actually a bit tricky to make the text look good on different screen sizes. I got a something semi-working for 1 image on the home page: https://github.com/mbyczkowski/lacocina/blob/3184b9dd45c2d395b4adec4c31e0c6b203ecc174/app/views/home/show.html.haml#L2-L7 and https://github.com/mbyczkowski/lacocina/blob/3184b9dd45c2d395b4adec4c31e0c6b203ecc174/app/assets/stylesheets/_home.scss#L34-L46

We could probably solve the responsiveness with a bunch of media queries. Also, a while back I discovered that units can be specified in % of viewport size - https://css-tricks.com/viewport-sized-typography/ and http://caniuse.com/#feat=viewport-units. So we could specify that a text is never bigger than half of the height or width of the device's screen. Not sure if that particular CSS feature is useful to us in this case, but still it seemed pretty fascinating to me 😉
